meraki/__init__.py

```diff
@@ -167,20 +170,21 @@ def __init__(self,
             use_iterator_for_get_pages=use_iterator_for_get_pages,
         )
 
+        self._enable_kwargs_validation = enable_kwarg_validation
```

Comment:
This should not be needed. None of the other config kwargs receive this treatment and we should not start treating kwargs this way.